### DISCUSSION TITLE
HSM support in Hydra

### DISCUSSION THREAD

**Discussion by lortabac** [ory/hydra#3215] [Q&A] [answered]
Is there a Hydra binary (or Docker image) with HSM support? I was only able to make the functionality work by building Hydra from the sources.
If it's not there yet, when can we expect this feature to be released?

**Comment by vinckr** [comment, marked as answer]
Hello, 
apologies for the late answer... but this is now supported in the quickstart docker-compose:  https://github.com/ory/hydra/blob/master/quickstart-hsm.yml
See also the documentation:  
https://www.ory.sh/docs/hydra/self-hosted/hsm-support
